actions/project.go

```diff
@@ -69,7 +97,99 @@ func DownloadTemplate(c *cli.Context) {
 	utils.DeleteTempFile(zipFileName)
 }
 
-//ValidateProject type
-func ValidateProject() {
-	//code here
+// ValidateProject returns the language and buildType for a project at given filesystem path
+func ValidateProject(c *cli.Context) {
+	projectPath := c.Args().Get(0)
```

Comment:
DownloadTemplate() has a check that the directory parameter is set. This function should have one too!
See: https://github.com/eclipse/codewind-installer/pull/70/files#diff-7cf8a504cf7756979e16701b4b2789b7R57